fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTabLayouts.kt

```diff
@@ -31,10 +36,18 @@ import org.mozilla.fenix.R
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
 import org.mozilla.fenix.compose.tabstray.TabGridItem
 import org.mozilla.fenix.compose.tabstray.TabListItem
+import org.mozilla.fenix.tabstray.browser.compose.DragItemContainer
+import org.mozilla.fenix.tabstray.browser.compose.createGridReorderState
+import org.mozilla.fenix.tabstray.browser.compose.createListReorderState
+import org.mozilla.fenix.tabstray.browser.compose.detectPressAndDragGestures
+import org.mozilla.fenix.tabstray.browser.compose.detectVerticalPressAndDrag
 import org.mozilla.fenix.tabstray.ext.MIN_COLUMN_WIDTH_DP
 import org.mozilla.fenix.theme.FirefoxTheme
 import kotlin.math.max
 
+const val SPAN_ITEM_KEY = "span"
+const val HEADER_ITEM_KEY = "header"
```

Comment:
maybe leave a comment here to mention that this is so the header can't be reordered?